packages/firestore/src/core/sync_engine.ts

```diff
@@ -1412,7 +1417,11 @@ export function loadBundle(
   syncEngineImpl.assertSubscribed('loadBundle()');
 
   // eslint-disable-next-line @typescript-eslint/no-floating-promises
-  loadBundleImpl(syncEngineImpl, bundleReader, task);
+  loadBundleImpl(syncEngineImpl, bundleReader, task).then(() => {
+    if (!syncEngineImpl.isPrimaryClient) {
```

Comment:
Is there a downside to invoking this on all tabs? Seems like we should issue this notification even on the primary tabs.

Reply:
You are right, this can be called on all tabs.